standards.md

```diff
@@ -0,0 +1,315 @@
+| async | (optional) An indicator that this module is executed asynchronously. Asynchronous modules should provide a status message on complete. |
+| long_name | A human readable name of module or configuration. |
+| description | A few sentences to describe the module or configuration. |
+| runtime | A list of either names or key-value tables specifying the executables and third-party libraries (modules, packages, ...) required to run all code in the module, specified as follows: <br> a. The names or keys in the list are the names of executable programs needed, in version-string format (described below). <br>b. The values associated with the keys are a list of third-party libraries for that language (as named in that language's library repository), again in version-string format. Only libraries not included as part of that language's standard installation need to be listed. <br>c. The format for version strings is one of '(name)', '(name)==(version)', '(name)>=(version)' or '(name)<=(version)'. Both (name) and (version) may be arbitrary strings (not containing =, < or >). |
+| selftest_script | (optional) This should be a UNIX command-line string, to be executed in the module’s code directory, which runs a script to perform any automated tests of the module’s code. <br>a. Module authors are encouraged to write unit and integration tests for their code in order to simplify debugging and maintenance. <br>b. The executable and any libraries required to run the script should be included in the runtime specification above. This includes UNIX shells. <br>c. The test script will run in the environment specified in “Runtime Environment” below, except for the model data features. In particular, the script may access files in the DEM’s code and observational data folders. <br>d. The test script should not assume the DEM has access to the example model data. <br>e. The test script should obey all the restrictions in the "Running the DEM" section. <br>f. The test script should return exit code 0 only if all tests pass, and != 0 otherwise. All output should be written to either stdout or stderr. |
+
+<p>&nbsp;</p>
+
+The varlist specifies the variables that are needed from the model data.  Each entry in the list requests one or more model datasets for a given variable.  Required varlist keys:
+| Key  | Definition |
+|------|------------|
+| var_name | This should be one of the CF (version 1.7) [standard names](http://cfconventions.org/standard-names.html), or an abbreviation used in CMOR 3.5 (see [CMIP6-CMOR Tables](https://github.com/PCMDI/cmip6-cmor-tables/tree/master/Tables) under key "variable_entry"). If the name exists the variable must be scalar-valued: for example, (u,v,w) wind velocity components should be requested as three separate entries. If the module requires non-standard data that can be computed from CF variables (e.g., column integrated water vapor), the module should perform that computation internally.  |
```

Comment:
What about core input variables that do not have CF definitions?

Reply:
Opened an issue for this question: https://github.com/Earth-System-Diagnostics-Standards/EMDS/issues/5

Reply:
@jkrasting Does MDTF have a solution in use for such variables?

Reply:
The MDTF `fieldlist*` files allow the framework to acknowledge non-CF variables.